client/gov_request_types.go

```diff
@@ -33,6 +34,7 @@ type GetProposalByIDRequest struct {
 
 type GetProposalByIDReply struct {
```

Comment:
I modified the struct's name and fields for 'olclient' command. Let's resolve our conflicts before merge.

Reply:
sure, inside this `GetProposalByIDReply`, I added `ProposalFunds` just to test if my withdraw works.
we can either use this one or come with a new one including proposal funds.

Reply:
I have a new one for this query. The olclient will allow either listing single proposal by ID or multiple proposals by state.

Reply:
cool, then we can use yours.